### DISCUSSION TITLE
suggestion for the next version of Sketchware.

### DISCUSSION THREAD

**Discussion by AkTr-Team** [Sketchware-Pro/Sketchware-Pro#579] [Ideas]
I wanted to make a suggestion.  Do you know Android 11?  So, the Blocks of Files that it contains in Sketchware, doesn't work for users of android 11, if you could fix it, besides me, several people would be very grateful... THANKS FOR CONTINUE WITH SKETCHWARE<3

**Comment by ilyassesalama** [comment]
Sketchware Pro allows to make new blocks anytime, you can make blocks that support Android 11's scoped storage then share it on our Discord server in #custom-blocks so everyone can use them when needed.

However, IMO; these blocks should be built-in since they're kind of a must-have.
